### ISSUE TITLE
consider moving the window keywords to nonReserved list

### ISSUE THREAD

**Issue by vinothchandar** [confluentinc/ksql#4773]
Follow up from a discussion in #4733 .. Should we move key words like `HOPPING`, `TUMBLING`, `RETENTION` etc to non reserved, so they can be used in user field names?

**Comment by mjsax** [comment]
I don't think this is a good idea. I would be rather restrictive. \cc @colinhicks @vcrfxia @agavra
